web/src/pages/index.ts

```diff
@@ -0,0 +1,4 @@
+// be sure to add all pages here
+
+export { default as Home } from "./Home";
+export { default as Login } from "./Login";
```

Comment:
Let's have a chat about the frontend folder structure so that we can be aligned in the approach.